docs/acm/component/server-foundation/metrics.md

```diff
@@ -6,12 +6,12 @@ Nomenclature:
 - `New` implies just added as a part of ACM 2.6 / ACM 2.7
```

Comment:
all foundation metrics are added in 2.7 except acm_managed_cluster_info, I think we should re-define `New` and the Status of foundation metrics